cmd/mr_list.go

```diff
@@ -109,6 +111,14 @@ func mrList(args []string) ([]*gitlab.MergeRequest, error) {
 		mrMilestone = milestone.Title
 	}
 
+	if mrReviewer != "" {
+		reviewerID, err := lab.UserIDByUserName(mrReviewer)
```

Comment:
It conflicts with the code from MR #650, where I'm removing the `lab.UserIDByUserName` function.
Use the `getUserID` from cmd/utils.go instead.